deploy/panorama_command.sh

```diff
@@ -31,8 +31,15 @@ docker build \
 command=""
 
 if [ $1 = "deploy" ]; then
-  echo "--- Deploying $DEPLOYMENT_APP_NAME"
-  command="git push heroku $BUILDKITE_COMMIT:master"
+  deploy_branch=""
+  if git ls-remote --exit-code --heads origin main > /dev/null; then
```

Comment:
Can we add a comment explaining how this command searches for the existence of main?